db/migrate/20210609145622_add_review_committee_fields_to_proposals.rb

```diff
@@ -0,0 +1,9 @@
+class AddReviewCommitteeFieldsToProposals < ActiveRecord::Migration[6.1]
+  def change
+    add_column :proposals, :research, :text
```

Comment:
Refactor (potentially blocking): Are all these fields optional, i.e. not null? Just wondering since in the model there is validation for presence.

Reply:
Yup: these fields are required. I was following the pattern of other fields in CFP-app and didn't put a nullable constraint on them in the database: field requirements are enforced only in the model in the upstream app (and exception being fields created by Devise).

I can't envision any issues with putting a null constraint on them, so I'll do that!